models/netsuite2/intermediate/int_netsuite2__acctxperiod_exchange_rate_map.sql

```diff
@@ -33,14 +34,12 @@ period_exchange_rate_map as ( -- exchange rates used, by accounting period, to c
   from consolidated_exchange_rates
 
   where consolidated_exchange_rates.to_subsidiary_id in (select subsidiary_id from subsidiaries where parent_id is null)  -- constraint - only the primary subsidiary has no parent
-  {% if var('netsuite2__multibook_accounting_enabled', true) %}
-    and consolidated_exchange_rates.accounting_book_id in (select accounting_book_id from accounting_books where is_primary)
-  {% endif %}
```

Comment:
Was there a reason this was removed?

Reply:
This logic was removing consolidated exchange rates for books where `is_primary` was false. I assume the intent was to prevent perceived duplicates. However, these were not duplicate records. The consolidated exchange rates can be different per book, and we should include the book in downstream joins.